nix/cardano-services/deployments.nix

```diff
@@ -0,0 +1,57 @@
+let
+  inherit (inputs.std) dmerge;
+  inherit (inputs.std.inputs) haumea;
+  inherit (inputs.std.lib.ops) readYAML;
+
+  loadYaml = _: _: readYAML;
```

Comment:
:raised_eyebrow:

Reply:
I kept the function signature that implements the `haumea` contract. So this is to say: "it's not an error, it's here so that the reader can identify and understand the haumea contract. In-lining makes this easily overlooked.

And you stumbling over it, it seems to have fulfilled its purpose :smile:

Reply:
Can you just drop the type definition comment or something so contract would be clear?

Reply:
Most certainly in this [force-push](https://github.com/input-output-hk/cardano-js-sdk/compare/80e4eae962d7b5d7f01bfd29aff952a87cff8e76..ad57dd41ec1b74dfea5df6acfe29630b9d49c24b)